quence.lua

```diff
@@ -952,3 +952,8 @@ function sleep(secs)
     repeat
     until os.clock() > ntime
 end
+
+function cleanup ()
```

Comment:
@millxing This guy doesn't actually do anything as-is; it's never being called.
In any case, we don't really have to exit clean, as norns basically "pulls out the rug" if the user changes scripts.
OTOH, it might be useful to have this fire when the sequencer is paused, so as to pause any MIDI devices which might be chasing our clock...
I don't really have a way to test it, so it would have to be up to you :)

Reply:
It does seem unnecessary, but some other scripts seem to have it (also uncalled). I think I read that it is called automatically when the script is exited, like init() is called when starting? Probably no harm in leaving it there.

Reply:
Ohhhh! I didn't know that; I probably should 😛

Reply:
For posterity: https://llllllll.co/t/norns-scripting/14120/136
Zebra sez: "if your script defines cleanup(), it will be executed by the script loading module when the next script is loaded."